README.md

```diff
@@ -7,916 +7,62 @@
 
 _Take control of your internal daemons!_
 
-**Pebble** helps you to orchestrate a set of local service processes as an organized set.
-It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can
-easily manage non-system processes independently from the system services, but it was
-designed with unique features that help with more specific use cases.
+**Pebble** is a lightweight Linux service manager that helps you orchestrate a set of local service processes as an organized set. It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services, but it was designed with unique features that help with more specific use cases.
```

Comment:
I am not 100% on this, and it's in the original text, but I believe that generally it's "well-known" when you're modifying an object and "well known" when you're using it standalone. So here it should be hyphenated.

Reply:
Agreed. This is already fixed in the docs (https://canonical-pebble.readthedocs-hosted.com/en/latest/), but let's fix it here too.

Let's also copy the other changes we've done in the new docs paragraph, eg "unique features such as layered configuration and an HTTP API that help with more specific use cases".